src/apprt/gtk-ng/class/surface.zig

```diff
@@ -863,6 +933,10 @@ pub const Surface = extern struct {
             priv.core_surface = null;
         }
 
+        if (priv.mouse_hover_url) |v| {
+            glib.free(@constCast(@ptrCast(v)));
+            priv.mouse_hover_url = null;
+        }
```

Comment:
Is there not zig equivalent to `g_clear_pointer()`?

Reply:
`glib.clearPointer` is the equivalent.

Reply:
> `glib.clearPointer` is the equivalent.

I don't think this will work Zig slices.